src/main/scala/com/fulcrumgenomics/testing/VariantContextSetBuilder.scala

```diff
@@ -92,7 +92,9 @@ class VariantContextSetBuilder(sampleNames: Seq[String] = List("Sample")) extend
                  genotypeAlleles: List[String] = List.empty,
                  genotypeAttributes: Map[String,Any] = Map.empty,
                  sampleName: Option[String] = None,
-                 phased: Boolean = false): this.type = {
+                 phased: Boolean = false,
+                 ad: Option[List[Int]] = None
```

Comment:
`ad` -> `alleleDepth` 

Avoid using one or two letter parameters in public APIs, and use them only in very small scopes.

Reply:
Why can we just have `alleleDepths: List[Int] = List.empty`?  What's the added value of an option versus an empty list?

Reply:
The allele depth is automatically set by the builder when you don't define it, so I thought it would be better as an `Option` to indicate that you have control, rather than it just being empty by default. 

I've changed it to a List.empty for now, but happy to change it back.